loleaflet/js/global.js

```diff
@@ -315,13 +315,13 @@
 			return this.decoder.decode(this.doSlice(bytes, start,end));
 		};
 		this.parseIncomingArray = function(arr) {
-//			console.debug('proxy: parse incoming array of length ' + arr.length);
+			//			console.debug('proxy: parse incoming array of length ' + arr.length);
 			for (var i = 0; i < arr.length; ++i)
 			{
 				var left = arr.length - i;
 				if (left < 4)
 				{
-//					console.debug('no data left');
+					//					console.debug('no data left');
```

Comment:
This doesn't seem right. It will look pretty weird when the `//` part is removed while debugging.